lib/grandpa/network.go

```diff
@@ -174,39 +174,43 @@ func (s *Service) sendMessage(msg GrandpaMessage) error {
 	return nil
 }
 
-func (s *Service) sendNeighbourMessage(interval time.Duration) {
+// notifyNeighbor will gossip a NeighbourMessage every 5 minutes, however we reset the ticker
+// whenever a finalisation occur meaning that a neighbour message already was sent by s.initiateRound()
+func (s *Service) notifyNeighbor(interval time.Duration) {
```

Comment:
can you move `interval` as an attribute of the `Service`. You can set it as `neighbourMessageInterval` in the constructor for `Service`.

Reply:
This `interval` is different, the `interval` in the `Service` is the amount of time we wait to send messages in the stages of the round (around 1 sec) and the interval in the `notifyNeighbor` is around 5 min and we should send a neighbor message every round change or at least 5 min following the docs